internal/storage/storage_handle.go

```diff
@@ -77,6 +94,18 @@ func NewStorageHandle(ctx context.Context,
 		}),
 		storage.WithPolicy(storage.RetryAlways))
 
-	sh = &storageClient{client: sc}
+	client := &storageClient{client: sc}
+	return client, err
+}
+
+func (sh *storageClient) BucketHandle(bucketName string) (bh *bucketHandle,
+	err error) {
+	storageBucketHandle := sh.client.Bucket(bucketName)
+	_, err = storageBucketHandle.Attrs(context.Background())
+	if err != nil {
+		return
```

Comment:
do you want the format the error to mention that specified bucket is not found

Reply:
Right now we get error message as "storage: bucket doesn't exist".